requirements.txt

```diff
@@ -15,3 +15,5 @@ PyYAML>=3.10.0 # MIT
 # NOTE(mattt): This causes issues w/ ansible's apt_key module and SNI
 #urllib3>=1.21.1 # MIT
 virtualenv>=13.1.0 # MIT
+ansible==2.4.3.0
```

Comment:
To avoid duplication could we not use the ansible that gets installed into the venv as part of the bootstrap-ansible.sh script? We could change that approach up.